pkg/ccl/serverccl/server_controller_test.go

```diff
@@ -181,6 +182,42 @@ VALUES($1, $2, $3, $4, $5)`, id, secret, username, created, expires)
 	require.Equal(t, body.Sessions[0].ApplicationName, "hello system")
 }
 
+func TestServerControllerMultiNodeTenantStartup(t *testing.T) {
```

Comment:
This test fails under stress on master any time we get a mismatched node ID vs sql instance ID.